zenml/core/backends/orchestrator/kubernetes/entrypoint.py

```diff
@@ -0,0 +1,69 @@
+#  Copyright (c) maiot GmbH 2020. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+"""Entrypoint for gcp orchestrator"""
+
+import base64
+import json
+import os
+import sys
+
+import fire
+
+from zenml.core.backends.orchestrator.gcp.orchestrator_gcp_backend import \
+    TAR_PATH_ARG
+from zenml.core.backends.orchestrator.local.orchestrator_local_backend import \
+    OrchestratorLocalBackend
+from zenml.core.repo.repo import Repository
+from zenml.core.standards import standard_keys as keys
+from zenml.utils import path_utils
+
+EXTRACTED_TAR_DIR_NAME = 'zenml_working.tar.gz'
+EXTRACTED_TAR_FILE_PATH = os.path.join(os.getcwd(), EXTRACTED_TAR_DIR_NAME)
+EXTRACTED_TAR_DIR = os.path.join(os.getcwd(), 'zenml_working')
+
+
+class PipelineRunner(object):
+
+    def run_pipeline(self, config_b64: str):
+        # Load config from base64 - stringified version
+        config = json.loads(base64.b64decode(config_b64.encode('utf-8')))
+
+        orch_args = config[keys.GlobalKeys.ENV][keys.EnvironmentKeys.BACKENDS][
+            OrchestratorLocalBackend.BACKEND_KEY][keys.BackendKeys.ARGS]
+
+        # overwrite the backend to do local execution.
+        config[keys.GlobalKeys.ENV][keys.EnvironmentKeys.BACKENDS][OrchestratorLocalBackend.BACKEND_KEY][
```

Comment:
This line is more than 80 characters -> Perhaps indent the third nesting to next line. PyCharm is not doing well with these cases